INA219.cpp

```diff
@@ -69,10 +69,17 @@ const uint8_t MODE1	= 0;
 INA219::INA219(t_i2caddr addr): i2c_address(addr) {
 }
 
-void INA219::begin() {
+bool INA219::begin() {
     Wire.begin();
+    if (!reset())
+    {
+      Serial.println("INA219 initialization failed.");
```

Comment:
No Serial.print in library

Reply:
Understood. Possibly return error value instead of true/false